source/includes/common-mc-admin-config.rst

```diff
@@ -0,0 +1,978 @@
+   part of the document body. Additional updates to that object modify the
+   existing table entry for that object. Similarly, deleting the object
+   also deletes the corresponding table entry.
+   
+``access``
+   For each bucket event, MinIO creates a JSON document with the event
+   details and appends it to the table with a MySQL-generated
+   random ID. Additional updates to an object result in new index entries, 
+   and existing entries remain unmodified.
+
+.. end-minio-notify-mysql-format
+
+.. start-minio-notify-mysql-max-open-connections
+
+Specify the maximum number of open connections to the MySQL database.
+
+Defaults to ``2``.
+
+.. end-minio-notify-mysql-max-open-connections
+
+.. start-minio-notify-mysql-queue-dir
+
+Specify the directory path to enable MinIO's persistent event store for
+undelivered messages, such as ``/home/events``.
+
+MinIO stores undelivered events in the specified store while the MySQL 
+server/broker is offline and replays the directory when connectivity resumes.
+
+.. end-minio-notify-mysql-queue-dir
+
+.. start-minio-notify-mysql-queue-limit
+
+Specify the maximum limit for undelivered messages. Defaults to ``10000``.
+
+.. end-minio-notify-mysql-queue-limit
+
+.. start-minio-notify-mysql-comment
+
+Specify a comment to associate with the MySQL configuration.
+
+.. end-minio-notify-mysql-comment
+
+
+.. Descriptions for Kafka bucket notification configurations.
+   Used in the following files:
+   - /source/reference/minio-server/minio-server.rst
+   - /source/reference/minio-cli/minio-mc-admin/mc-admin-config.rst
+   - /source/monitoring/bucket-notifications/publish-events-to-kafka.rst
+
+.. start-minio-notify-kafka-enable
+
+Specify ``on`` to enable publishing bucket notifications to a Kafka
+service endpoint.
+
+Defaults to ``off``.
+
+.. end-minio-notify-kafka-enable
+
+.. start-minio-notify-kafka-brokers
+
+Specify a comma-separated list of Kafka broker addresses. For example:
+
+``"kafka1.example.com:2021,kafka2.example.com:2021"``
+
+.. end-minio-notify-kafka-brokers
+
+.. start-minio-notify-kafka-topic
+
+Specify the name of the Kafka topic to which MinIO publishes 
+bucket events.
+
+.. end-minio-notify-kafka-topic
+
+.. start-minio-notify-kafka-sasl-username
+
+Specify the username for performing SASL/PLAIN or SASL/SCRAM authentication
+to the Kafka broker(s).
+
+.. end-minio-notify-kafka-sasl-username
+
+.. start-minio-notify-kafka-sasl-password
+
+Specify the password for performing SASL/PLAIN or SASL/SCRAM authentication
+to the Kafka broker(s).
+
+.. end-minio-notify-kafka-sasl-password
+
+.. start-minio-notify-kafka-sasl-mechanism
+
+Specify the SASL mechanism to use for authenticating to the Kafka broker(s).
+MinIO supports the following mechanisms:
+
+- ``PLAIN`` (Default)
+- ``SCRAM``
+
+.. end-minio-notify-kafka-sasl-mechanism
+
+.. start-minio-notify-kafka-tls-client-auth
+
+Specify the client authentication policy of the Kafka broker(s).
```

Comment:
@poornas @vadmeste I could not find anything on what the expectation for this field actually is, nor did the kafka or go library docs shed much light here either.

Reply:
cc @Praveenrajmani @kannappanr on the above

Reply:
You can point it to this https://golang.org/pkg/crypto/tls/#ClientAuthType @ravindk89 That is actually an Enum.

Reply:
Great - I saw that, but wasn't honestly sure what it would do - but if we can point at that and call it done I've got no objections.